thrust/detail/complex/catrig.h

```diff
@@ -601,7 +602,12 @@ complex<double> catanh(complex<double> z)
   y = z.imag();
   ax = fabs(x);
   ay = fabs(y);
-  
+
+  // MSVC needs to pull this in from the std namespace
+#if THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_MSVC
+  using std::atanh;
+#endif
```

Comment:
I think we probably ought to try adding a math.h include instead, or just try removing this now that the complex header properly includes math.h.

Reply:
Let's open an issue and do this as a separate commit to avoid blocking this change from landing.

Reply:
Ah apparently `thrust/detail/complex/c99math.h` does the equivalent of

```
namespace thrust::detail::complex { using ::atanh; }
```

for every platform but MSVC. Adding this on MSVC fixes the compile error and the complex tests are building and passing locally. I'll push the new PR in a sec.